vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -75,11 +78,14 @@ func main() {
 	vpaLister := vpa_api_util.NewAllVpasLister(vpaClient, make(chan struct{}))
 	kubeClient := kube_client.NewForConfigOrDie(config)
 	factory := informers.NewSharedInformerFactory(kubeClient, defaultResyncPeriod)
+	if *allowToAdjustLimits {
+		factoryForLimitsChecker = factory
```

Comment:
I'd rather explicitly pass the no-op limitChecker when *allowToAdjustLimits = false